licenses.d2iq.yaml

```diff
@@ -1,6 +1,7 @@
 ignore:
   - ghcr.io/mesosphere/dkp-container-images/docker.io/mesosphere/grafana-plugins:v0.0.1-d2iq.0
   - docker.io/mesosphere/trivy-bundles:0.53.0-20240726T101319Z
+  - gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
```

Comment:
If the container dependency is removed correctly, then there is no need for adding this to ignore.